docs/manual/inference.tex

```diff
@@ -229,6 +228,31 @@
 
 You may need to wait a few minutes for the command to complete.
 
+\subsectionAndLabel{Whole-program inference results}{wpi-results}
+
+The result of invoking \<wpi.sh> are log files stored in the \<dljc-out/> folder
+of the target project.
+The \<dljc-out/> folder contains:
+
+\begin{itemize}
```

Comment:
I suggest re-ordering this list in terms of importance/usefulness for debugging. In my experience, the most useful files are:
* `typecheck.out` (for cases when WPI runs; especially useful for debugging crashes that happen on every iteration of WPI)
* `build_output.txt` (for cases when the build file isn't working, which happens a lot)
* `wpi-stdout.log` (for cases when there is a problem with WPI, such as an ephemeral crash or a crash on an early iteration)
* the various `dljc-stdout-*` files (for cases when DLJC is doing something wrong)
* everything else